contracts/protocol/integration/auction-price/BoundedStepwiseLogarithmicPriceAdapter.sol

```diff
@@ -55,7 +55,7 @@ contract BoundedStepwiseLogarithmicPriceAdapter {
         uint256 timeBucket = _timeElapsed / bucketSize;
 
         // Protect against logarithmic argument overflow
-        if (timeBucket > type(uint256).max / timeCoefficient) {
+        if (timeBucket > uint256(type(int256).max) / timeCoefficient) {
```

Comment:
Add a test for the case that this code protects against?

Reply:
Yes thank you @FlattestWhite , will add in base branch before merging to main